packages/blocks/src/page-block/edgeless/toolbar/shape-tool/shape-tool-button.ts

```diff
@@ -69,13 +69,6 @@ export class EdgelessShapeToolButton extends LitElement {
     super.firstUpdated(changedProperties);
   }
 
-  updated(changedProperties: Map<string, unknown>) {
```

Comment:
I like using less `updated` callback, the `changedProperties.has('prop')` pattern is really bad for refactoring.